enterprise/frontend/src/metabase-enterprise/whitelabel/index.js

```diff
@@ -16,113 +17,223 @@ import {
-          display_name: t`Font`,
-          widget: FontWidget,
-        },
-        {
-          key: "application-font-files",
-          widget: FontFilesWidget,
-          getHidden: settings => settings["application-font-files"] == null,
-        },
-        {
-          key: "application-colors",
-          display_name: t`Color Palette`,
-          widget: ColorSettingsWidget,
-        },
-        {
-          key: "application-logo-url",
-          display_name: t`Logo`,
-          type: "string",
-          widget: LogoUpload,
-        },
-        {
-          key: "application-favicon-url",
-          display_name: t`Favicon`,
-          type: "string",
-        },
-        {
-          key: "landing-page",
-          display_name: t`Landing Page`,
-          type: "string",
-          placeholder: "/",
-          widget: LandingPageWidget,
-        },
-        {
-          key: "loading-message",
-          display_name: t`Loading message`,
-          type: "select",
-          options: getLoadingMessageOptions(),
-          defaultValue: "doing-science",
-        },
-        {
-          key: "help-link",
-          display_name: t`Help Link in the Settings menu`,
-          description: (
-            <p>
-              {t`The Settings menu includes a Help link that goes to `}
-              <Anchor href="https://www.metabase.com/help">{t`this page`}</Anchor>
-              {t` by default.`}
-            </p>
-          ),
-          widget: HelpLinkSettings,
-          defaultValue: "metabase",
-        },
-        {
-          key: "show-metabot",
-          display_name: t`Metabot greeting`,
-          description: null,
-          type: "boolean",
-          widget: MetabotToggleWidget,
-          defaultValue: true,
-        },
-        {
-          key: "show-lighthouse-illustration",
-          display_name: t`Lighthouse illustration`,
-          description: null,
-          type: "boolean",
-          widget: LighthouseToggleWidget,
-          defaultValue: true,
-        },
-        {
-          key: "show-metabase-links",
-          display_name: t`Documentation and references`,
-          description: <MetabaseLinksToggleDescription />,
-          widget: MetabaseLinksToggleWidget,
-        },
-      ],
+  PLUGIN_ADMIN_SETTINGS_UPDATES.push(
+    sections => ({
+      ...sections,
+      whitelabel: {
+        name: t`Appearance`,
+        key: "-redirect-to-branding",
+        settings: [
+          {
+            key: "-redirect-to-branding",
+            widget: () => (
+              <RedirectWidget to="/admin/settings/whitelabel/branding" />
+            ),
+          },
+        ],
+      },
+      "whitelabel/branding": {
+        tabs: [
+          {
+            name: t`Branding`,
+            key: "branding",
+            to: "/admin/settings/whitelabel/branding",
+            isActive: true,
+          },
+          {
+            // eslint-disable-next-line no-literal-metabase-strings -- Admin settings
```

Comment:
Let me think about this. The rule for disabling the rule for the whole file is forbidden. My intention would be to have everyone leave exactly the comment at the point of disabling to tell why that would be disabled. I imagined there would be cases where we want to disable the rule on certain lines but not others and for different reasons. But that doesn't seem to be the case so far.

Reply:
On second thought, I like not letting things slip through the cracks. I'd like to prohibit using the whole file disable comment in favor for inline disable comments since I don't think we'll have to do this much often. Having to add clear reason in every line forces us to think (I trust you guys) about the reason why it should be disabled.